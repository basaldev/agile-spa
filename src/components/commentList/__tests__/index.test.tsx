import * as React from 'react';
import { shallow } from 'enzyme';
import { CommentList } from '../index';

describe('CommentList', () => {
  test('render CommentList', () => {
      const wrapper = shallow(
        <CommentList comments={[{ username: 'username', content: 'content' }]} />
      );
      expect(wrapper).toMatchSnapshot();
  });
});
