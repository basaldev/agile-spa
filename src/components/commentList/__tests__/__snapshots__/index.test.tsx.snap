// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentList render CommentList 1`] = `
<styled.table>
  <tbody>
    <styled.tr
      key="0"
    >
      <styled.th>
        <Typography
          content="username"
          type="p"
        />
      </styled.th>
      <styled.td>
        <Typography
          content="content"
          type="p"
        />
      </styled.td>
    </styled.tr>
  </tbody>
</styled.table>
`;
